### ISSUE TITLE
Keyboard navigation in youtube not working even when youtube.com excluded

### ISSUE THREAD

**Issue by skobylite** [televator-apps/vimari#221]
The keyboard navigation keys to skip forward/back by 10s (J,K,L) aren't working in vimari, even when youtube.com is excluded in the config file.

**Comment by sooheon** [comment]
I'm seeing similar behavior where Vimari interferes with ESC and CMD-u (and possibly other keys) in https://roamresearch.com, even when the site is excluded. Maybe even with exclusion Vimari still intercepts keypresses and doesn't pass them through correctly?

**Comment by ayroblu** [comment]
Yeah I'm seeing this too, fyi, you can jump into "insert mode" by pressing `i` by default, but that's not ideal cause you have to do that every time

**Comment by flybayer** [comment]
I have the same problem on GitHub
